frame/contracts/src/lib.rs

```diff
@@ -97,6 +97,8 @@ mod wasm;
 
 pub mod chain_extension;
 pub mod migration;
+#[cfg(feature = "unsafe-debug")]
```

Comment:
Move it to the module please. Just a style nit but looks nicer in my opinion.

Reply:
moved